model-optimizer/mo/utils/cli_parser.py

```diff
@@ -153,7 +153,7 @@ def readable_dir(path: str):
     :return: path if the directory is readable
     """
     if not os.path.isdir(path):
-        raise Error('The "{}" is not existing directory'.format(path))
+        raise Error('The "{}" directory does not exist'.format(path))
```

Comment:
please do not change the error message. Due to this change mo_params_tests.mo_args_test.NegativeInputs.test_nonexisiting_input_model is failing now.

Reply:
Removed.